schemas/src/config/document.parameter.yaml

```diff
@@ -66,23 +74,49 @@ properties:
     type: object
     # VS Code only
     markdownDescription: |
-      > [Online Documentation][01]
+      ***
+      [_Online Documentation_][01]
+      ***
 
       Defines a set of key-value pairs for the parameter. This metadata isn't validated.
 
-      [01]: https://learn.microsoft.com/en-us/powershell/dsc/reference/schemas/config/parameter?view=dsc-3.0&preserveView=true#metadata-1
+      [01]: <DOCS_BASE_URL>/reference/schemas/config/parameter?<DOCS_VERSION_PIN>#metadata-1
     defaultSnippets:
-      - label:       New Metadata Property
-        description: Defines a key-value pair for the metadata
-        body:        { '${1:metadataName}': '$2' }
-      - label:       New metadata property (object)
-        description: Defines a new key-value pair for the metadata where the value is an object.
+      - label: ' New metadata property'
+        markdownDescription: |-
+          Defines a key-value pair for the metadata:
+
+          ```yaml
+          metadataName: value
+          ```
+        body:
+          ${1:metadataName}: ${2:value}
+
+      - label: ' New metadata property (object)'
+        markdownDescription: |-
+          Defines a new key-value pair for the metadata where the value is an object.
+
+          ```yaml
+          metadataName:
+            key: value
+          ```
         body:
-          '${1:metadataName}': { '${2:key}': '${3:value}' }
-      - label:       New metadata property (array)
-        description: Defines a new key-value pair for the metadata where the value is an array.
+          ${1:metadataName}:
+            ${2:key}: ${3:value}
+
+      - label: ' New metadata property (array)'
```

Comment:
@SteveL-MSFT - the extra whitespace is intentional, it moves them to the top of the IntelliSense completion list - otherwise they often end up hidden at the bottom, depending on the extensions you have installed.